charts/umbrella/README.md

```diff
@@ -467,16 +469,34 @@ helm install -f values-adopter-portal.yaml umbrella . --namespace umbrella --cre
 
 #### Data exchange
 
-:warning: Please be aware of [Note for R24.05](#note-for-r2405)
-
 Involved components:
 
-EDC, MIW, DTR, Vault (data provider and consumer in tx-data-provider), CentralIdP.
+> DBRS-Server-Memory, Data Consumer, Data Provider, SSI Dim Wallet Stub.
```

Comment:
I noticed that if would makes sense to enable pgadmin4 also for the data exchange (in case somebody want to look in the databases), like for the portal., could you please do that?
```suggestion
- Data Provider
- Data Consumer 1
- Data Consumer 2 (Optional)
- BPN-DID-Resolution-Service (BDRS)
- SSI DIM Wallet Stub
- pgadmin4
```